### PR TITLE
MAJ-1156_update copyright issues for all documentation module in all Geomajas projects

### PR DESCRIPTION
**Pull request by streulma**

### CODE REVIEW

**Comment by janvenstermans** [comment]
Matthias asks: is this ok: chaninging "licenseName: MIT Copyright (C) 2011-2014 by Jorik Tangelder (Eight Media)" (first file)

File: NOTICE.txt [hunk 1]
```diff
@@ -8,5 +8,5 @@ GWT
     licenseUrl: http://www.apache.org/licenses/LICENSE-2.0.html
 
 hammer.js
-    licenseName: MIT Copyright (C) 2011-2014 by Jorik Tangelder (Eight Media)
+    licenseName: MIT Copyright (C) 2011-2015 by Jorik Tangelder (Eight Media)
```
Comment (janvenstermans) [inline NOTICE.txt]: Change back to "licenseName: MIT Copyright (C) 2011-2014 by Jorik Tangelder (Eight Media)"

**Comment by janvenstermans** [comment]
Jan Venstermans: -1

**Comment by oliverm-be** [comment]
Oliver May: +1 (binding)

**Comment by jandm** [comment]
-1 (binding)

**Comment by janvenstermans** [comment]
See correction in https://github.com/geomajas/geomajas-project-hammer-gwt/pull/8

**Pull request closed without merging**